### PR TITLE
Feature/rdsssam 155 messages output from the native samvera format should be translated

### PR DESCRIPTION
**Pull request by blackrat**
Messaging and integration with rdss_cdm with versioning for schema validation etc.

### CODE REVIEW

File: willow/app/controllers/concerns/hyrax/notifications/notifiers.rb [hunk 1]
```diff
@@ -11,23 +11,15 @@ module Notifiers
         # end
```
Comment (fmcc) [inline willow/app/controllers/concerns/hyrax/notifications/notifiers.rb]: Do we want to leave this file in just now or will we save removing this notification code for a subsequent purge ticket?

File: willow/config/metadata_request.yml [hunk 2]
```diff
@@ -1,118 +1,130 @@
...
-#  fileStorageLocation:
-#  fileStoragePlatform:
-#    storagePlatformUuid:
-#    storagePlatformName:
-#    storagePlatformType:
-#    storagePlatformCost:
+2.0.0: &latest
+  create_and_update: &create_and_update
+    # NOTE: This document actually directly controls how the message is generated. Adding a new or removing an item needs
+    # to be done in conjunction with the edge
```
Comment (fmcc) [inline willow/config/metadata_request.yml]: With the edge? Might be worth adding a bit more info to this comment.

File: willow/lib/hyrax/notifications/senders/approve.rb [hunk 3]
```diff
@@ -1,28 +1,14 @@
 module Hyrax
   module Notifications
```
Comment (fmcc) [inline willow/lib/hyrax/notifications/senders/approve.rb]: Again, do we want to get rid of this file as well - I see you've pulled out `build_message.rb` as well

File: willow/app/controllers/concerns/hyrax/notifications/notifiers.rb (hunk 1 above)
Comment (blackrat) [reply willow/app/controllers/concerns/hyrax/notifications/notifiers.rb]: Either way. Might be best to do a purge/clean-up soon though.

File: willow/config/metadata_request.yml (hunk 2 above)
Comment (blackrat) [reply willow/config/metadata_request.yml]: Ah. Yes. It's the endpoint classes. I'll rework that and make it clearer.

File: willow/lib/hyrax/notifications/senders/approve.rb (hunk 3 above)
Comment (blackrat) [reply willow/lib/hyrax/notifications/senders/approve.rb]: Yes. This one should be ok to be removed now too.

File: willow/app/controllers/concerns/hyrax/notifications/notifiers.rb (hunk 1 above)
Comment (blackrat) [reply willow/app/controllers/concerns/hyrax/notifications/notifiers.rb]: Are we also going to remove Articles, Books, Datasets, RdssDatasets and Images? This is included in all of them. Possibly better to leave it until after the Kinesis changes are more Whispered.

Comment (fmcc) [reply willow/app/controllers/concerns/hyrax/notifications/notifiers.rb]: Yeah, we'd talked about raising a ticket to go through and purge cottage labs code that we weren't using. Best to leave for that.

File: willow/lib/hyrax/notifications/senders/approve.rb (hunk 3 above)
Comment (blackrat) [reply willow/lib/hyrax/notifications/senders/approve.rb]: Actually. It appears to be also called from the other controllers. I'm going to leave it for the clean-up ticket.

**Pull request merged**